git.go

```diff
@@ -0,0 +1,89 @@
+package scopeagent
+
+import (
+	"bufio"
+	"github.com/google/uuid"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+type GitData struct {
+	Repository		string
+	Commit			string
+	SourceRoot		string
+	Branch			string
+}
+
+type GitDiff struct {
+	Type			string	`json:"type"`
+	Version			string  `json:"version"`
+	Uuid			string  `json:"uuid"`
+	Files			[]DiffFileItem  `json:"files"`
+}
+type DiffFileItem struct {
+	Path			string  `json:"path"`
+	Added			int		`json:"added"`
+	Removed 		int		`json:"removed"`
+	Status			string  `json:"status"`
+}
+
+// Gets the current git data
+func GetCurrentGitData() *GitData {
+	repoBytes, _ := exec.Command("git", "remote", "get-url", "origin").Output()
```

Comment:
We cannot assume `git` is available

Reply:
do you prefer to handle the error in case not available, or do you want me to read the `.git` folder manually?

Reply:
Now I'm handling the errors when calling git to not fail.